Tasks/MavenV3/maventask.ts

```diff
@@ -565,4 +568,36 @@ function execBuildWithRestore() {
         execBuild();
     }
 }
+
+/**  function read code coverage report as DOM */
+function readCodeCoverageReportAsDom(): jsdom.JSDOM {
+    const htmlString: string = fs.readFileSync(path.join(reportDirectory, 'index.html'), 'utf-8');
```

Comment:
Could it make sense to move 'index.html' out as parameter?

Reply:
Done.